### PR TITLE
feature/PWA: installable , offline mode

### PR DESCRIPTION
**Pull request by Sravan2305**
PWA functionality

### CODE REVIEW

File: public/ofline.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
```
Comment (MiKr13) [inline public/ofline.html:1]: Rename this file to offline.html please.

File: src/serviceWorker.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export const service_worker = ()=>{
+    if ("serviceWorker" in navigator) {
+        navigator.serviceWorker.register("./sw.js").then(() => {
+          //Service Worker Registration Succesful
```
Comment (MiKr13) [inline src/serviceWorker.ts:4]: Service worker registeration can be notified in console as it's more of a convention so please put
```js
console.log('Service Worker Registered');
```
Also handle .catch case for errors!

**Review by MiKr13 — CHANGES_REQUESTED** [review]
Please implement appropriate changes

**Comment by alok722** [comment]
LGTM

**Review by alok722 — APPROVED** [review]
Look goods, merging

**Pull request merged**